lib/symlink/prepare.js

```diff
@@ -24,8 +23,10 @@ function prepareSymlink(folderResolver, optResolver) {
     var writePath = path.resolve(basePath, file.relative);
 
     // Wire up new properties
+    // TODO new stats will have invalid props, but target stats will have the
+    // wrong ones...
     file.stat = (file.stat || new fs.Stats());
-    file.stat.mode = mode;
+    file.stat.mode = LINK_MODE;
```

Comment:
If we update the `fs.Stat` metadata after write in `.dest()` (don't remember), do we also do it for `symlink()`

Reply:
We don't do it in either case (currently). Reason is probably we can't write stats for a symlink (but we could split the file-operations to make the reflecting the vinyl stats off disk part separately accessible)?

Reply:
I think that would be beneficial from the perspective of doing the same thing (or similar, in this case) for each operation.